book/internals/setters.nim

```diff
@@ -0,0 +1,76 @@
+import nimib, nimibook
+
+nbInit(theme = useNimibook)
+
+nbText: """
+## **Setters**
+Setters are a convenience mechanism for widgets.
+Through them you can hide fields on your `WidgetState`.
+Parent-widgets will be able to assign to "pseudo-fields", whose assigned values get converted and then forwarded to the actual field on `WidgetState`.
+
+allow parent-widgets to assign to fields on `WidgetState` with different types of values that get converted to the field on `WidgetState`!
+
+Every setter consists of:
+- a `setter <PseudoFieldName>` declaration with the type that the setter receives, e.g. `setter margins: int` defines a setter that will receive an int from a parent-widget
```

Comment:
I would just write SetterName instead of PseudoFieldName. (applies to all instances of PseudoFieldName).